apps/events/migrations/0013_create_event_categories.py

```diff
@@ -23,4 +23,4 @@ def create_event_categories(apps, schema_editor):
             ]
         )
 
-    operations = [migrations.RunPython(create_event_categories)]
+    # operations = [migrations.RunPython(create_event_categories)]
```

Comment:
Do not modify automatically generated files
If you want to make changes or do a rollback, create a new migration for tihs